src/k2/module/k23si/utils.h

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
The contents of this file are metric-specific. We have some metric stuff in the Prometheus.h header in transport. please place this there.

In general, try not to use general names such as "utils" or "common" for stuff. Conversely, if you should come across some code in a common/utils file and you have an idea of how to name it better, do feel free to make the change :)

Reply:
done